hello-pepr-alias/package.json

```
@@ -30,7 +30,7 @@
     }
   },
   "dependencies": {
-    "pepr": "^0.38.3"
+    "pepr": "file:../pepr-0.0.0-development.tgz"
```

```suggestion
    "pepr": "^0.38.3"
```
Due to the way the dev CLI is now changing the `package.json` file (as a way to swap in an under-test version of pepr), when a test run fails / early-exits, _this dev value_ will be swapped into the `package.json/package-lock.json`.

You're going to have to check for this & specifically _not_ include these in your PRs.